### DISCUSSION TITLE
tables: can I un-set a sort?

### DISCUSSION THREAD

**Discussion by dajones537** [uswds/uswds#5421] [General]
I'd like to be able to re-set my table to zero, back to initial state. I see there is an internal unsetSort function.
![image](https://github.com/uswds/uswds/assets/123194829/785d2eb2-161a-4316-ac43-345e09c27850)
But I don't think I can manually call this, far as I'm aware. Anyone know a good way to reset sort? Thanks.

**Comment by dajones537** [comment]
The case in which I need to do this is where the table is already empty, so I decided to just do this:

    const headers = document.querySelectorAll('th');
    headers.forEach((header: HTMLElement) => {
        if (header.hasAttribute('aria-sort')) {
            header.removeAttribute('aria-sort');
        }
    })

If I actually had content in my table then I don't believe this would un-sort, but that's okay, since my use case is just when the table has just been reset to empty.

**Comment by sknep** [comment]
This is an interesting question. If the table has data and someone wanted to reset sort, would it need to return to the original sort order? Right now, we don't keep that around....
